Only in bdb-6.2.23-1: db-6.2.23
diff -ru bdb-6.2.23/dist/configure bdb-6.2.23-1/dist/configure
--- bdb-6.2.23/dist/configure	2016-03-29 08:45:49
+++ bdb-6.2.23-1/dist/configure	2024-07-25 13:53:13
@@ -19380,13 +19380,13 @@
               class TLSClass2 {
               public: static  $ax_tls_decl_keyword int tlsvar;
               };
-              template<typename T>  $ax_tls_defn_keyword  T* TLSClass<T>::tlsvar = NULL;
+              template<typename T>  $ax_tls_defn_keyword  T* TLSClass<T>::tlsvar = 0;
               $ax_tls_defn_keyword int TLSClass2::tlsvar = 1;
               static $ax_tls_decl_keyword int x = 0;
 int
 main ()
 {
-TLSClass<int>::tlsvar = NULL; TLSClass2::tlsvar = 1;
+TLSClass<int>::tlsvar = 0; TLSClass2::tlsvar = 1;
   ;
   return 0;
 }
diff -ru bdb-6.2.23/src/dbinc/atomic.h bdb-6.2.23-1/src/dbinc/atomic.h
--- bdb-6.2.23/src/dbinc/atomic.h	2016-03-29 08:45:54
+++ bdb-6.2.23-1/src/dbinc/atomic.h	2024-07-25 13:49:02
@@ -70,7 +70,9 @@
  * These have no memory barriers; the caller must include them when necessary.
  */
 #define	atomic_read(p)		((p)->value)
+#if ! defined(__cplusplus) /* atomic_init is invalid in c++ */
 #define	atomic_init(p, val)	((p)->value = (val))
+#endif
 
 #ifdef HAVE_ATOMIC_SUPPORT
 
