diff -ru db-6.2.23.NC/src/dbinc/atomic.h bdb-6.2.23-fix/src/dbinc/atomic.h
--- db-6.2.23.NC/src/dbinc/atomic.h	2016-03-28 15:45:54.000000000 -0400
+++ bdb-6.2.23-fix/src/dbinc/atomic.h	2020-03-24 23:31:16.000000000 -0400
@@ -70,7 +70,7 @@
  * These have no memory barriers; the caller must include them when necessary.
  */
 #define	atomic_read(p)		((p)->value)
-#define	atomic_init(p, val)	((p)->value = (val))
+#define	atomic_init_bdb(p, val)	((p)->value = (val))
 
 #ifdef HAVE_ATOMIC_SUPPORT
 
@@ -225,7 +225,7 @@
 #define	atomic_dec(env, p)	(--(p)->value)
 #define	atomic_compare_exchange(env, p, oldval, newval)		\
 	(DB_ASSERT(env, atomic_read(p) == (oldval)),		\
-	atomic_init(p, (newval)), 1)
+	atomic_init_bdb(p, (newval)), 1)
 #else
 #define	atomic_inc(env, p)	__atomic_inc_int(env, p)
 #define	atomic_dec(env, p)	__atomic_dec_int(env, p)
Only in bdb-6.2.23-fix/src/dbinc: atomic.h-e
diff -ru db-6.2.23.NC/src/mp/mp_fget.c bdb-6.2.23-fix/src/mp/mp_fget.c
--- db-6.2.23.NC/src/mp/mp_fget.c	2016-03-28 15:45:54.000000000 -0400
+++ bdb-6.2.23-fix/src/mp/mp_fget.c	2020-03-24 23:31:16.000000000 -0400
@@ -654,7 +654,7 @@
 
 		/* Initialize enough so we can call __memp_bhfree. */
 		alloc_bhp->flags = 0;
-		atomic_init(&alloc_bhp->ref, 1);
+		atomic_init_bdb(&alloc_bhp->ref, 1);
 #ifdef DIAGNOSTIC
 		if ((uintptr_t)alloc_bhp->buf & (sizeof(size_t) - 1)) {
 			__db_errx(env, DB_STR("3025",
@@ -969,7 +969,7 @@
 			MVCC_MPROTECT(bhp->buf, mfp->pagesize,
 			    PROT_READ);
 
-		atomic_init(&alloc_bhp->ref, 1);
+		atomic_init_bdb(&alloc_bhp->ref, 1);
 		MUTEX_LOCK(env, alloc_bhp->mtx_buf);
 		alloc_bhp->priority = bhp->priority;
 		alloc_bhp->pgno = bhp->pgno;
Only in bdb-6.2.23-fix/src/mp: mp_fget.c-e
diff -ru db-6.2.23.NC/src/mp/mp_mvcc.c bdb-6.2.23-fix/src/mp/mp_mvcc.c
--- db-6.2.23.NC/src/mp/mp_mvcc.c	2016-03-28 15:45:54.000000000 -0400
+++ bdb-6.2.23-fix/src/mp/mp_mvcc.c	2020-03-24 23:31:16.000000000 -0400
@@ -281,7 +281,7 @@
 #else
 	memcpy(frozen_bhp, bhp, SSZA(BH, buf));
 #endif
-	atomic_init(&frozen_bhp->ref, 0);
+	atomic_init_bdb(&frozen_bhp->ref, 0);
 	if (mutex != MUTEX_INVALID)
 		frozen_bhp->mtx_buf = mutex;
 	else if ((ret = __mutex_alloc(env, MTX_MPOOL_BH,
@@ -440,7 +440,7 @@
 #endif
 		alloc_bhp->mtx_buf = mutex;
 		MUTEX_LOCK(env, alloc_bhp->mtx_buf);
-		atomic_init(&alloc_bhp->ref, 1);
+		atomic_init_bdb(&alloc_bhp->ref, 1);
 		F_CLR(alloc_bhp, BH_FROZEN);
 	}
 
Only in bdb-6.2.23-fix/src/mp: mp_mvcc.c-e
diff -ru db-6.2.23.NC/src/mp/mp_region.c bdb-6.2.23-fix/src/mp/mp_region.c
--- db-6.2.23.NC/src/mp/mp_region.c	2016-03-28 15:45:54.000000000 -0400
+++ bdb-6.2.23-fix/src/mp/mp_region.c	2020-03-24 23:31:16.000000000 -0400
@@ -278,7 +278,7 @@
 			     MTX_MPOOL_FILE_BUCKET, 0, &htab[i].mtx_hash)) != 0)
 				return (ret);
 			SH_TAILQ_INIT(&htab[i].hash_bucket);
-			atomic_init(&htab[i].hash_page_dirty, 0);
+			atomic_init_bdb(&htab[i].hash_page_dirty, 0);
 		}
 
 		mtx_base = mtx_prev = MUTEX_INVALID;
@@ -332,7 +332,7 @@
 		    DB_MUTEX_SHARED, &hp->mtx_hash)) != 0)
 			return (ret);
 		SH_TAILQ_INIT(&hp->hash_bucket);
-		atomic_init(&hp->hash_page_dirty, 0);
+		atomic_init_bdb(&hp->hash_page_dirty, 0);
 #ifdef HAVE_STATISTICS
 		hp->hash_io_wait = 0;
 		hp->hash_frozen = hp->hash_thawed = hp->hash_frozen_freed = 0;
Only in bdb-6.2.23-fix/src/mp: mp_region.c-e
diff -ru db-6.2.23.NC/src/mutex/mut_method.c bdb-6.2.23-fix/src/mutex/mut_method.c
--- db-6.2.23.NC/src/mutex/mut_method.c	2016-03-28 15:45:54.000000000 -0400
+++ bdb-6.2.23-fix/src/mutex/mut_method.c	2020-03-24 23:31:16.000000000 -0400
@@ -501,7 +501,7 @@
 	MUTEX_LOCK(env, mtx);
 	ret = atomic_read(v) == oldval;
 	if (ret)
-		atomic_init(v, newval);
+		atomic_init_bdb(v, newval);
 	MUTEX_UNLOCK(env, mtx);
 
 	return (ret);
Only in bdb-6.2.23-fix/src/mutex: mut_method.c-e
diff -ru db-6.2.23.NC/src/mutex/mut_tas.c bdb-6.2.23-fix/src/mutex/mut_tas.c
--- db-6.2.23.NC/src/mutex/mut_tas.c	2016-03-28 15:45:54.000000000 -0400
+++ bdb-6.2.23-fix/src/mutex/mut_tas.c	2020-03-24 23:31:16.000000000 -0400
@@ -47,7 +47,7 @@
 
 #ifdef HAVE_SHARED_LATCHES
 	if (F_ISSET(mutexp, DB_MUTEX_SHARED))
-		atomic_init(&mutexp->sharecount, 0);
+		atomic_init_bdb(&mutexp->sharecount, 0);
 	else
 #endif
 	if (MUTEX_INIT(&mutexp->tas)) {
@@ -643,7 +643,7 @@
 			F_CLR(mutexp, DB_MUTEX_LOCKED);
 			/* Flush flag update before zeroing count */
 			MEMBAR_EXIT();
-			atomic_init(&mutexp->sharecount, 0);
+			atomic_init_bdb(&mutexp->sharecount, 0);
 		} else {
 			DB_ASSERT(env, sharecount > 0);
 			MEMBAR_EXIT();
Only in bdb-6.2.23-fix/src/mutex: mut_tas.c-e
