diff -Nru test_0.0/configure xx/configure
--- test_0.0/configure	     1969-12-31 19:00:00.000000000 -0500
+++ xx/configure	     2019-04-04 14:22:40.000000000 -0400
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+src=`dirname $0`
+
+cat << EOF >> Makefile
+all:
+
+install:
+	echo Installing from $src to \$(DESTDIR)/usr/local/boost-1.69.0
+	mkdir -p \$(DESTDIR)/usr/local/boost-1.69.0
+	./${src}/bootstrap.sh --prefix=\$(DESTDIR)/usr/local/boost-1.69.0
+	./${src}/b2 install threading=multi,single link=shared,static -j12
+
+EOF
