diff -Nru test_0.0/configure xx/configure
--- test_0.0/configure	     1969-12-31 19:00:00.000000000 -0500
+++ xx/configure	     2020-05-28 14:22:40.000000000 -0400
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+src=`dirname $0`
+
+cat << EOF >> Makefile
+all:
+
+install:
+	echo Installing from $src to \$(DESTDIR)/\$(PREFIX)
+	echo Current directory $PWD
+	mkdir -p \$(DESTDIR)/\$(PREFIX)
+	mv ${src}/* .
+	./bootstrap.sh --prefix=\$(DESTDIR)/\$(PREFIX)
+	./b2 --prefix=\$(DESTDIR)/\$(PREFIX) toolset=clang-darwin link=static cxxflags='-Wall -g -O2 -fPIC -fvisibility=hidden' cflags='-Wall -g -O2 -fPIC -fvisibility=hidden' install
+
+EOF
