diff -ru fontconfig-2.13.1-orig/test/test-bz106632.c fontconfig-2.13.1/test/test-bz106632.c
--- fontconfig-2.13.1-orig/test/test-bz106632.c	2018-08-29 05:55:14.000000000 -0400
+++ fontconfig-2.13.1/test/test-bz106632.c	2020-03-12 22:10:20.000000000 -0400
@@ -154,18 +154,6 @@
     return ret;
 }
 
-FcChar8 *
-FcLangNormalize (const FcChar8 *lang)
-{
-    return NULL;
-}
-
-FcChar8 *
-FcConfigHome (void)
-{
-    return NULL;
-}
-
 int
 main (void)
 {
