diff -Nru hwloc-orig/Makefile.am hwloc-2.9.3/Makefile.am
--- hwloc-orig/Makefile.am      2023-10-17 11:00:48
+++ hwloc-2.9.3/Makefile.am     2023-10-17 11:00:53
@@ -23,7 +23,7 @@
 if BUILD_NETLOC
 SUBDIRS += netloc
 endif
-SUBDIRS += utils tests contrib/systemd contrib/completion contrib/misc contrib/hwloc-ps.www
+#SUBDIRS += utils tests contrib/systemd contrib/completion contrib/misc contrib/hwloc-ps.www
 # We need doc/ if HWLOC_BUILD_DOXYGEN, or during make install if HWLOC_INSTALL_DOXYGEN.
 # There's no INSTALL_SUBDIRS, so always enter doc/ and check HWLOC_BUILD/INSTALL_DOXYGEN there
 SUBDIRS += doc
 
