diff -ru openssl-1.1.1l/crypto/rand/rand_unix.c openssl-1.1.1l-1/crypto/rand/rand_unix.c
--- openssl-1.1.1l/crypto/rand/rand_unix.c	2021-08-24 09:38:47.000000000 -0400
+++ openssl-1.1.1l-1/crypto/rand/rand_unix.c	2021-11-24 14:42:14.000000000 -0500
@@ -35,7 +35,7 @@
 # include <sys/param.h>
 #endif
 #if defined(__APPLE__)
-# include <CommonCrypto/CommonRandom.h>
+# include <CommonCrypto/CommonCryptor.h>
 #endif
 
 #if defined(OPENSSL_SYS_UNIX) || defined(__DJGPP__)
Only in openssl-1.1.1l-1/crypto/rand: rand_unix.c~
