diff -ruN serf-1.3.10/Makefile serf-1.3.10-1/Makefile
--- serf-1.3.10/Makefile	1970-01-01 12:00:00
+++ serf-1.3.10-1/Makefile	2024-07-25 11:55:57
@@ -0,0 +1,17 @@
+ifeq ($(MACOSX_DEPLOYMENT_TARGET),)
+  ADDCC=
+else
+  ## scons is broken and will override MACOSX_DEPLOYMENT_TARGET so have to force it...
+  ARCH=$(shell uname -m)
+  ADDCC=CC='clang -arch $(ARCH) -mmacosx-version-min=$(MACOSX_DEPLOYMENT_TARGET)'
+endif
+
+all:
+	curl -L http://prdownloads.sourceforge.net/scons/scons-local-4.1.0.post1.tar.gz | tar fxz -
+	python3 scons.py $(ADDCC) PREFIX=/$(PREFIX) APR=/$(PREFIX)/bin/apr-1-config APU=/$(PREFIX)/bin/apu-1-config APR_STATIC=true CFLAGS='-fPIC $(CFLAGS)'
+
+install:
+	for i in lib include lib/pkgconfig; do if [ ! -e "$(DESTDIR)/$(PREFIX)/$$i" ]; then mkdir -p "$(DESTDIR)/$(PREFIX)/$$i"; fi; done
+	cp -p libserf-1.a '$(DESTDIR)/$(PREFIX)/lib/'
+	cp -p serf-1.pc '$(DESTDIR)/$(PREFIX)/lib/pkgconfig'
+	cp -p serf.h serf_bucket_types.h serf_bucket_util.h '$(DESTDIR)/$(PREFIX)/include/'
diff -ruN serf-1.3.10/SConstruct serf-1.3.10-1/SConstruct
--- serf-1.3.10/SConstruct	2023-05-18 21:33:55
+++ serf-1.3.10-1/SConstruct	2024-07-25 11:55:57
@@ -68,7 +68,7 @@
   default_libdir='..'
   default_prefix='Debug'
 else:
-  default_incdir='/usr'
+  default_incdir='$PREFIX'
   default_libdir='$PREFIX/lib'
   default_prefix='/usr/local'
 
@@ -426,7 +426,7 @@
                                                        env.get('GSSAPI_LIBS', '')),
                            })
 
-env.Default(lib_static, lib_shared, pkgconfig)
+env.Default(lib_static, pkgconfig)
 
 if CALLOUT_OKAY:
   conf = Configure(env)
@@ -456,8 +456,7 @@
                                      % (target_install_shared_path,
                                         install_shared_path)))
 
-env.Alias('install-lib', [install_static, install_shared,
-                          ])
+env.Alias('install-lib', [install_static])
 env.Alias('install-inc', env.Install(incdir, HEADER_FILES))
 env.Alias('install-pc', env.Install(os.path.join(libdir, 'pkgconfig'),
                                     pkgconfig))
