diff -ruN iodbc-42.6/configure iodbc-42.6-inst/configure
--- iodbc-42.6/configure	1970-01-01 12:00:00.000000000 +1200
+++ iodbc-42.6-inst/configure	2021-09-08 17:40:02.000000000 +1200
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+## figure out the source directory by using $0
+cwd=`pwd`
+cfg=`pwd`/$0
+cd `dirname $cfg`
+src=`pwd`
+cd $cwd
+
+echo Installing from $src, generating Makefile
+
+cat > Makefile <<EOF
+HEADERS = isql.h isqlext.h isqltypes.h sql.h sqlext.h sqlucode.h sqltypes.h odbcinst.h iodbcunix.h iodbcinst.h iodbcext.h
+SRC = $src
+
+all:
+install:
+	if [ ! -e "\$(DESTDIR)/\$(PREFIX)/include" ]; then mkdir -p "\$(DESTDIR)/\$(PREFIX)/include"; fi
+	for i in \$(HEADERS); do cp -p \$(SRC)/iodbc/include/\$\$i "\$(DESTDIR)/\$(PREFIX)/include/"; done
+EOF
